### DISCUSSION TITLE
land cover

### DISCUSSION THREAD

**Discussion by paula-lelis** [UMEP-dev/UMEP#404] [General]
Hello, I would like to know if UMEP accepts a land cover generated from a shapefile. The trees and buildings heights are necessary in this file?

**Comment by biglimp** [comment]
The land cover should be i raster format but could ofcourse be generated from vector data. I suggest you have a look in the tutorials on our webpage to see what kind of land cover that is required for the model you are interested in running.

**Reply by paula-lelis** [reply]
Hello, biglimp, thenk you very much for your answer. I would like to know if the LCZ converter also generates valid files for "land/ground cover" used for SOLWEIG.

**Comment by biglimp** [comment]
No, its too large scale

**Reply by paula-lelis** [reply]
Ok, thank you. I want to generate a Tmr map for an entire city, that´s why I was wondering about using LCZ maps. Do you know if solweig is capable to calculate that?

**Reply by biglimp** [reply]
It is possible but you need to divide in tiles, otherwise you will have memory issues. Preferred modelling domain size is around 1000*1000 grid points but you can probably do 4000*4000 if you have a very powerful computer.

**Reply by paula-lelis** [reply]
Ok, I will do that. Thank you very much!

**Reply by paula-lelis** [reply]
My machine is an Intel i7,  16GB ram, ssd 1T. Do you have an idea which modelling domain size could run in this machine? The city I am modelling is São Paulo.
